data/remnant missions.txt

```diff
@@ -110,13 +179,12 @@ mission "Remnant: Defense 1"
 event "remnant defense timer"
 	clear "Remnant defense delay"
 
-
-
+			
 mission "Remnant: Defense 2"
 	landing
 	source "Caelian"
 	name "Remnant bounty"
-	description "A Korath ship is flying around Remnant territory. Hunt it down and destroy it, then return to <planet> for payment."
+	description "A pack of Korath ships is flying around Remnant territory. Hunt it down and destroy it, then return to <planet> for payment."
```

Comment:
Thanks! I changed it to "them" and it sounds better to me to. 

Just thinking about it, I think if there was more emphasis on the pack (like giving it a name so it is a specific pack), then the it would sound better, but as is it sounds more like it is a random collection of ships that just happen to be a pack, so referring to the plural ships moreso than the pack itself sounds better.